### PR TITLE
PGAdmin documentation

### PR DESCRIPTION
**Pull request by AdityaMSaraf**
Added description on how to use the pgAdmin utility as another way to access the database in developer documentation.

### CODE REVIEW

File: developer/devDetails.html [hunk 1]
```diff
@@ -83,6 +83,22 @@ <h2 id="codeChange">Getting a running OED to recognize code changes</h2>
 			Windows</a>, it is important to do this properly.</p>
 
 	<h2 id="dbAccess">Accessing the database</h2>
+	<p>There are two ways to access the Postgres database that OED uses. Through the docker container or through pgAdmin
```
Comment (huss) [inline developer/devDetails.html:86]: Maybe add a colon (:) at the end of the line.

File: developer/devDetails.html [hunk 2]
```diff
@@ -83,6 +83,22 @@ <h2 id="codeChange">Getting a running OED to recognize code changes</h2>
...
+			</li>
+			<li>
+				You can also use the pgAdmin utility to access the database. For this method, you need to install pgAdmin on your device. The link to get the latest version is <a href="https://www.pgadmin.org/download/">here</a>.
+				Once you have downloaded and installed the correct version for your Operating System, you need to enable the port for the connection to pgAdmin. This can be done by going to the dockercompose.yml file and 
+				uncommenting the lines <code>ports:</code> and <code>-"5432":"5432"</code>. You should now be able to open pgAdmin and access the database on localhost port 5432. 
+			</li>
+		</ul>
+	</p>
+
 	<p>You can access the Postgres database that OED uses within the docker container. In the database docker container
```
Comment (huss) [inline developer/devDetails.html]: This now duplicates the bullet above. I propose to remove this paragraph. Then add to the end of the second bullet that this directly accesses the OED database as noted in first bullet.

**Review by huss — CHANGES_REQUESTED** [review]
Thanks to @AdityaMSaraf for this contribution. I have made a few comments to address.

File: developer/devDetails.html (hunk 1 above)
Comment (AdityaMSaraf) [reply developer/devDetails.html:86]: Added

File: developer/devDetails.html (hunk 2 above)
Comment (AdityaMSaraf) [reply developer/devDetails.html]: Removed paragraph following the bullets and added information to first two bullets

**Review by huss — APPROVED** [review]
Thanks to @AdityaMSaraf  for the changes. They were okay but I decided to enhance the directions somewhat. With this, the PR is ready to merge.

**Pull request merged**